File: BAEKJOON/java/5565_영수증.java
import java.io.*;

public class Main {
    public static int sum;
    public static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    public static BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

    public static void main(String[] args) throws IOException {
        input();
        solution();

        bw.flush();
        bw.close();
    }

    private static void input() throws IOException {
       sum = Integer.parseInt(br.readLine());
    }

    private static void solution() throws IOException {
        int price;

        for (int i = 0; i < 9; i++) {
            price = Integer.parseInt(br.readLine());
            sum -= price;
        }

        bw.write(String.valueOf(sum));
    }
}

File: BAEKJOON/cpp/5565_영수증.cpp
#include <iostream>
using namespace std;

int sum;

void input() {
	cin >> sum;
}

void solution() {
	int price;

	for (int i = 0; i < 9; i++) {
		cin >> price;
		sum -= price;
	}

	cout << sum;
}

int main() {
	ios::sync_with_stdio(false);
	cin.tie(NULL);
	cout.tie(NULL);

	input();
	solution();

	return 0;
}